setup.py

```diff
@@ -26,17 +26,17 @@
     ],
     packages=find_namespace_packages(exclude=["tests"]),  # Required
     install_requires=[
-        "PyYAML>=5.2",
-        "imageio>=2.5",
-        "marshmallow>=3.6.0,<4.0",
+        "numpy",
         "marshmallow-jsonschema",
         "marshmallow-union",
+        "marshmallow>=3.6.0,<4.0",
         "requests",
```

Comment:
This one shouldn't be in the pyodide
```suggestion
        "requests;platform_system!='Emscripten'",
```